subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/EclipsePlugin.java

```diff
@@ -275,6 +275,13 @@ public void execute(GenerateEclipseClasspath task) {
                 });
             }
         });
+
+        project.getPlugins().withType(JavaTestFixturesPlugin.class, new Action<JavaTestFixturesPlugin>() {
```

Comment:
Don't you want this instead? (Lazy task evaluation):

```suggestion
        project.getPlugins().withType(JavaTestFixturesPlugin.class).configureEach(new Action<JavaTestFixturesPlugin>() {
```

Reply:
That might make sense, but it should not cause any trouble as all the other configurations in this class are done the same way (wo/`configureEach`).

Reply:
This is about plugins. Not tasks. It should not matter which one you use. I would keep it as it is to be consisten.

I think it is confusing that the lazy methods were added to all domain object container. Many of them won't need them. The plugin container is one example.